tsconfig.rollup.json

```diff
@@ -1,6 +1,11 @@
 {
-	"extends": "./tsconfig.json",
+	"extends": "./tsconfig.build.json",
+	"compilerOptions": {
+		"noEmit": false,
+		"outDir": "dist",
+		"rootDir": "./"
+	},
```

Comment:
I doesn't seem to change any compiler option compared to the extends

Reply:
The subtle, but important inclusion change is required:
```
"./src/**/*",
```
because the main `tsconfig.json` file includes all sources (demo folder, too!):
```
"**/src/**/*",
```

Reply:
```
"outDir": "dist",
"rootDir": "./",
```
can be moved to `tsconfig.json`

Reply:
I'm not sure to understand, I'm talking about the compiler options only, which are the same in tsconfig.rollup.json and tsconfig.build.json

Reply:
I did this because:
vscode relies on a `tsconfig.json`. You just cannot configure the name of the main TypesScript config file vscode should use for whatever reasons. vscode's eslint support automatically takes into account what is defined in `tsconfig.json`.

rollup's plugin-typescript picks up automatically `tsconfig.json` (default) and therefore used demo source, too. That's why I introduced the other config.

But, I realized this is not the best way. I do it differently now. All tsconfigs are reverted. I kept the tsconfig entry in `rollup.config.ts` to make aware which tsconfig is used.
I introduced a `tsconfig.eslint.json` and added parserOptions to the eslint config.

Honestly, configuration of eslint + TypeScript is unpleasant and full of unexpected behaviour.